lib/Backend/LinearScan.cpp

```diff
@@ -1373,6 +1373,11 @@ LinearScan::FillBailOutRecord(IR::Instr * instr)
         if (currentRegionType == RegionTypeTry || currentRegionType == RegionTypeCatch || currentRegionType == RegionTypeFinally)
         {
             bailOutInfo->bailOutRecord->ehBailoutData = this->currentRegion->ehBailoutData;
+            if (this->func != instr->m_func)
+            {
+                bailOutInfo->bailOutRecord->ehBailoutData = NativeCodeDataNew(func->GetNativeCodeDataAllocator(), Js::EHBailoutData, this->currentRegion->ehBailoutData);
```

Comment:
Seems like an overkill to allocate new EHBailoutData objects (here and below) if all we want to track is whether the bailout from an EH region happened in an inlinee or not. Could add a bit on the interpreter or add a new flag in InterpreterStackFrameFlags or something similar.

Reply:
I added it on the InterpreterStackFrameFlags.